todolist.html

```diff
@@ -32,5 +32,57 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+        <div>
+            <p>Add Task</p>
+            <div>
+                <label>Name</label>
+                <div>100</div>
+                <input type='text'>
+                <small>Maximum of 100 characters.</small>
+            </div>
+            <div>
+                <label>Expected date of completion</label>
+                <input type='text'>
+                <small></small>
```

Comment:
What does this empty tag do?

Reply:
This small tag will be filled with text to inform the user when they have failed to fill in a required field (date is one of the required fields), but otherwise it is just empty. I thought it would be easier to leave an empty tag here and change the text content, than to create and append a small tag every time an error is made

Reply:
If this does not currently happen (the filling of the element when a user has failed to fill in a required field) then please remove these empty tags. Only add them when they are being used for something.